react-app/src/features/faq/LegacyFaq.tsx

```diff
@@ -18,6 +19,15 @@ export const LegacyFaq: React.FC<LegacyFaqProps> = () => {
   const { id } = useParams<{ id: string }>();
 
   const [openItems, setOpenItems] = useState<string[]>([]);
+
+  const expandAll = useCallback(() => {
+    const allIds = [];
+    oneMACFAQContent.forEach(({ qanda }) => {
+      qanda.forEach(({ anchorText }) => allIds.push(anchorText));
+    });
```

Comment:
Also, this could be a `.flatMap`:

```ts
const allIds = oneMACFAQContent.flatMap(({ qanda }) => 
  qanda.map(({ anchorText }) => anchorText)
);
```

This should work as well